### PR TITLE
Awesome Book with ES6

### PR DESCRIPTION
**Pull request by Htetaungkyaw71**
# Awesome book with ES6

- Added modules in the project file
- Created separated Script files in the modules
- Changed Arrow functions
- Used `let` and `const`
- Fixed Linters errors 
- Added .gitignore file
- Setup npm packages
- Changed file name to index.js, index.css 
- Updated Readme file

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  checkls(library);
+  spa();
+});
+
+const time = DateTime.now().toFormat("LLLL dd'th' tt");
+const date = document.querySelector('.date');
+
+date.innerHTML = time;
+
+library.books.forEach((book) => library.render(book));
```
Comment (Bondok6) [inline index.js]: - [ ] Great job here 💯 but there is a small problem, when the page is reloaded the data will disappear, to fix this problem you have to get the data from the `localStorage` and render it 👍

> 💡 I can see that you already have a function called `checkls(library);` doing that, Great 🙌
> So you just have to render books inside the  `document.addEventListener('DOMContentLoaded', () => {})`
> For Example 💡
```js
document.addEventListener('DOMContentLoaded', () => {
  checkls(library);
  library.books.forEach((book) => library.render(book));
  spa();
});
```

File: index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+document.addEventListener('DOMContentLoaded', () => {
+  checkls(library);
+  spa();
+});
+
+const time = DateTime.now().toFormat("LLLL dd'th' tt");
+const date = document.querySelector('.date');
+
+date.innerHTML = time;
```
Comment (Bondok6) [inline index.js]: - [OPTIONAL] I noticed that the time here is static, I suggest to make it dynamic by using `setInterval()` method 

> For Example 💡

```js
setInterval(() => {
  const time = DateTime.now().toFormat("LLLL dd'th' tt");
  const date = document.querySelector('.date');
  date.innerHTML = time;
}, 1000);
```

File: README.md [hunk 3]
```diff
@@ -1 +1,75 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- ### Original design idea by **Cindy Shin in Behance**, in association with the Creative Commons [license](./CC.md) of the design.
+
+## 📝 License
+
+This project is [MIT](..Licence) licensed.
```
Comment (Bondok6) [inline README.md]: - [OPTIONAL] The MIT link is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @Htetaungkyaw71 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No Linter errors ✅
- Correct GitHub flow ✅
- Your repo is professional ✅
- Great job using ES6 Features ✅


## Required Changes ♻️
- [ ] _Check the comments under the review._ ⬇

## Optional suggestions
- [ ] _Check the comments under the review._ ⬇

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Htetaungkyaw71,

✔️ Books persist after page refresh

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**